chi/context.py

```diff
@@ -16,11 +21,13 @@
 
 LOG = logging.getLogger(__name__)
 
+DEFAULT_SITE = "CHI@UC"
```

Comment:
I see this is unused. I think I envisioned automatically running `set_site` if no site was set. Maybe `check_credentials` should set this if the environment variable is unset before hand? Jupyterhub automatically sets this however, but users external to Jupyter might not, which may impact later calls to the hardware module for example.